### PR TITLE
Create tooling.md

### PR DESCRIPTION
**Pull request by celestehorgan**

### CODE REVIEW

**Review by justaugustus — CHANGES_REQUESTED** [review]
@celestehorgan -- Tool eval is within the scope of WS Community, so maybe it want to send requests for eval to inclusivenaming@ and/or ws-community@ instead?

Ideally, we have an issue template to request demo time and point folks to that?
Thoughts here, @inclusivenaming/leads?

**Comment by justaugustus** [comment]
@ddavisjones -- for input as well.

**Comment by celestehorgan** [comment]
@justaugustus – maybe ws-community@ instead of leads?

**Comment by savage-alex** [comment]
Looks good!!!!

**Comment by jasonbrooks** [comment]
I think this looks good. 

+1 to having an issue template to request demo time.

File: ws-oss/tooling.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Awesome Inclusive Language Tooling
+
+The following is a list of tools **not maintained by the Inclusive Naming Initiative** which can scan or lint for inclusive langugage concerns. 
+
+Tools included in this document must be demo'ed to the Inclusive Naming Initiative. PRs to this list should include a demonstration date discussed with
+the Inclusive Naming Initiative leads – please email leads@inclusivenaming.org to contact them.
```
Comment (celestehorgan) [inline ws-oss/tooling.md:6]:
```suggestion
the Inclusive Naming Community workstream – please email ws-community@inclusivenaming.org to contact them.
```

**Pull request merged**